.github/workflows/cla.yml

```diff
@@ -0,0 +1,56 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]

```

Comment:
Is it really required for issues or only for PRs?

Reply:
I agree and took it out 👍